app/routes.js

```diff
@@ -7,5 +7,14 @@ router.get('/', function (req, res) {
 })
 
 // add your routes here
+router.post('/check-state-pension-options-1', function (req, res) {
+  if (req.body.loginoption === "government-gateway") {
+    res.redirect('https://www.tax.service.gov.uk/gg/sign-in?continue=%2Fcheck-your-state-pension%2Faccount&origin=nisp-frontend&accountType=individual')
```

Comment:
This isn't a pattern you see often, a form redirecting you to another domain, I wonder if there's any possibility for this to confuse users.